packages/pulp/pulp.spec

```diff
@@ -474,28 +474,30 @@ Pulp provides replication, access, and accounting for software repositories.
 %{_usr}/lib/tmpfiles.d/
 %endif
 # 640 root:apache
-%defattr(640,root,apache,-)
+%defattr(640,apache,pulp,-)
```

Comment:
@dkliban , why do you change user as well and not leave root here? and just make it accessible for pulp group instead of apache?

Reply:
this is an error with search and replace. I'll update it to root,pulp